micrometer-test/src/main/java/io/micrometer/core/tck/MeterRegistryCompatibilityKit.java

```diff
@@ -379,6 +381,18 @@ void record() {
                     () -> assertEquals(0.01, sample.duration(TimeUnit.MICROSECONDS)),
                     () -> assertEquals(1, t.activeTasks()));
 
+            assertThat(t.measure()).satisfiesExactlyInAnyOrder(
+                    measurement -> assertThat(measurement).satisfies(m -> {
+                        assertThat(m.getValue()).isEqualTo(1.0);
+                        assertThat(m.getStatistic()).isSameAs(ACTIVE_TASKS);
+                    }),
+                    measurement -> assertThat(measurement).satisfies(m -> {
+                        assertThat(m.getValue()).isEqualTo(TimeUtils.convert(10, TimeUnit.NANOSECONDS, t.baseTimeUnit()));
+                        assertThat(m.getStatistic()).isSameAs(DURATION);
+                    })
+            );
+            sample.stop();
```

Comment:
Is this needed? Looks like the sample is stopped 3 lines below.

Reply:
😨